Apps/Mock/src/Controllers/PhsaController.cs

```diff
@@ -36,14 +36,29 @@ public class PhsaController : ControllerBase
         /// <param name="limit">The limit on the number of records returned.</param>
         /// <returns>The mocked laboratory order json.</returns>
         [HttpGet]
-        [Route("laboratoryOrder")]
+        [Route("LaboratoryOrders")]
         [Produces("application/json")]
         public ContentResult LaboratoryOrders([FromQuery] string subjectHdid, [FromQuery] string limit)
         {
             string? payload = AssetReader.Read("HealthGateway.Mock.Assets.LaboratoryOrders.json");
```

Comment:
The content in this file will need to be updated to match the data that comes back from PHSA's `/api/v1/Lab/Plis/LabSummary`.